docroot/themes/custom/mass_theme/templates/paragraphs/paragraph--org-section-long-form--default.html.twig

```diff
@@ -16,6 +16,14 @@
   "externalSidebar": true
 } %}
   {% block stackedRowContentOverride %}
+    {# Add a wrapper for stat component to align with other components. #}
+    {% set statComponent = "" %}
+    {% for org_component in org_components %}
+      {% set statComponent = org_component[0].field_stat ? "true" : "false" %}
```

Comment:
This check is not working as intended, this class is being added to all rows
<img width="1695" alt="Screen Shot 2021-11-22 at 5 03 12 PM" src="https://user-images.githubusercontent.com/5789411/142941901-d5df127d-c9cd-4e50-85bb-eb83346b8c83.png">

@Javi-er have done similar fixes before, I will @ him on the ticket to chat about the solution.